symfony/client/messages.en_US.json

```diff
@@ -185,6 +185,16 @@
           "source": "No employees match the selected filters",
           "target": "No employees match the selected filters",
           "description": ""
+        },
+        "employee": {
+          "source": "Employee",
+          "target": "Employee",
+          "description": "Staff currently working at your company."
```

Comment:
The description is basically for the translator I guess. We may not need this for FE. @samanthajayasinghe what do you think